src/app/inventory/subclass.ts

```diff
@@ -62,10 +62,3 @@ export function getDamageTypeForSubclassPlug(item: PluggableInventoryItemDefinit
   }
   return null;
 }
-
-/** Get the DamageType enum value for a subclass item definition. */
-export function getDamageTypeForSubclassDef(subclass: DestinyInventoryItemDefinition) {
-  return subclass.inventory?.bucketTypeHash === BucketHashes.Subclass
-    ? subclass.talentGrid?.hudDamageType
-    : undefined;
-}
```

Comment:
I think this would mean that the change you made the other day would now be unnecessary @bhollis. If we go full loadout item resolution this isn't needed anymore though.

Reply:
Sure as long as it isn't used anywhere.

Reply:
Build would be complaining if so, which means this is safe